monkey/infection_monkey/exploit/exploiter_wrapper.py

```diff
@@ -24,18 +24,21 @@ def __init__(
             telemetry_messenger: ITelemetryMessenger,
             event_queue: IEventQueue,
             agent_binary_repository: IAgentBinaryRepository,
+            servers: List[str],
```

Comment:
```suggestion
            servers: Sequence[str],
```

Reply:
Servers need to be passed to the exploiter every time the exploiter is run. The list of servers may change over time.